AISKU/scripts/publishAzImgToCdn.ps1

```diff
@@ -107,18 +117,23 @@ if ([string]::IsNullOrWhiteSpace($global:connectDetails.sasToken) -eq $true) {
 
 Write-Log "======================================================================"
 
-$releaseFiles = GetReleaseFiles 
-if ($null -eq $releaseFiles -or $releaseFiles.Count -eq 0) {
+$releaseFiles, $releaseNightFiles = GetReleaseFiles 
+$actualCount = $releaseFiles.Count + $releaseNightFiles.Count
+$versions = $versionlist -split ","
+if ($null -eq $releaseFiles -and $null -eq $releaseNightFiles) {
     Write-LogFailure "Unable to find any release files"
+} elseif ($actualCount -ne 3*$versions.Count) {
+    Write-LogWarning "Files number is incorrect. Expected: $($versions.Count*3), Actual: $actualCount"
 }
 
 Write-Log "Release Files : $($releaseFiles.Count)"
-
+Write-Log "Release Night Files : $($releaseNightFiles.Count)"
 Write-Log "----------------------------------------------------------------------"
 
 $contentDisposition = "inline"
 
 # Publish the img to the folder that is same to the script folder.
 PublishFiles $releaseFiles "scripts/b" $cacheControl $contentType $overwrite $contentDisposition
+PublishFiles $releaseNightFiles "nightly" $cacheControl $contentType $overwrite $contentDisposition
```

Comment:
Why call out nightly specifically???

This can be abstracted out as the "container" which can be "dev", "nightly", "next", "beta" or "tst" which is what this portion of the main publishing script is doing https://github.com/microsoft/ApplicationInsights-JS/blob/db897fe314e0fc8d09f6b2768979b2e0d33c8c99/AISKU/scripts/publishReleaseToCdn.ps1#L125-L148...

This script "should" be able to handle any of these without needing to call out and add specific collections like lines 73-77

Reply:
because when I passed in the versionlist, the version is a string and does not have type

Reply:
I see. I thought the type was passed in. Turns out that Get-VersionDetails could assign the type.